core/src/main/scala/stryker4s/run/ProcessMutantRunner.scala

```diff
@@ -65,10 +65,9 @@ class ProcessMutantRunner(command: Command, process: ProcessRunner)(implicit con
     MutantRunResults(runResults, calculateMutationScore(totalMutants, detected.size), duration)
   }
 
-  private[this] def runMutant(mutant: Mutant, workingDir: File, subPath: Path): MutantRunResult = {
-    val id = mutant.id
-    info(s"Starting test-run ${id + 1}...")
-    process(command, workingDir, ("ACTIVE_MUTATION", id.toString)) match {
+  private[this] def runMutant(mutant: Mutant, workingDir: File, subPath: Path, id: Int): MutantRunResult = {
+    info(s"Starting test-run $id...")
+    process(command, workingDir, ("ACTIVE_MUTATION", (id - 1).toString)) match {
```

Comment:
Does this work with the correct mutation? The mutation is made based on the mutant id, not the index that's given here.

So if 4 mutants are generated with ids 0, 1, 2, 3, and 2 is excluded, then the `ACTIVE_MUTATION`s set here would be 0, 1, 2